### PR TITLE
[yaml/hu] add Hungarian YAML translation

### PR DESCRIPTION
**Pull request by ditam**
A Hungarian translation of the YAML article, no new content added

### CODE REVIEW

**Comment by geoffliu** [comment]
Thanks!

File: hu-hu/yaml-hu.html.markdown [hunk 1]
```diff
@@ -0,0 +1,146 @@
+---
+language: yaml
+filename: learnyaml-hu.yaml
+contributors:
+  - ["Adam Brenecki", "https://github.com/adambrenecki"]
+translators:
+  - ["Tamás Diószegi", "https://github.com/ditam"]
```
Comment (geoffliu) [inline hu-hu/yaml-hu.html.markdown]: Could you add a `lang: hu-hu`  here?

**Comment by geoffliu** [comment]
@ditam Thanks for the translation. Could you just add the language label, and we can merge this.

**Comment by ditam** [comment]
@geoffliu Done!

**Comment by geoffliu** [comment]
@ditam Merged!

**Pull request merged**